### DISCUSSION TITLE
Feature Request: "Internal note" for Speaker

### DISCUSSION THREAD

**Discussion by navlisilvan** [pretalx/pretalx#1744] [Ideas] [answered]
Hi,

There is a nice "Internal note" (Interner Vermerk) Input field in the Backend for Talks.
It would be nice to also have such a Field for Speaker to share infos about the speaker between people in the programme planning team and to document them for further ecents. 

What do you think?

Thanks a lot!

**Comment by badbadc0ffee** [comment]
I like the idea at first glance.

On the other hand, we actually need even more. We’ve created a plugin that lets us write "speaker care messages" meant for urgent notifications — for example, notes we need to follow up on when a speaker checks in at the venue. These notes are no longer relevant after the event.

At the same time, there are definitely things I’d like to keep track of for the *next* event — but that aren’t urgent at all during the current one.

Some of this information can also be very sensitive, so access should probably be limited separately.

All in all, my suggestion would be to handle this kind of thing via a plugin, so it can better meet the specific needs of individual events.

**Reply by rixx** [reply]
I think both things are true: for a more in-depth process, a plugin is probably the right call, but for a smaller / less involved event, just having the comment field on the speaker profile would already help a bunch. I think the biggest question is if the comment should be used by and/or visible to review users with no further permissions.

**Comment by rixx** [comment, marked as answer]
Good idea! Feature request accepted – now tracking at https://github.com/pretalx/pretalx/issues/2031
